### DISCUSSION TITLE
A javascript error occurred in the main process

### DISCUSSION THREAD

**Discussion by rajil** [dani-garcia/vaultwarden#2347] [Q&A]
I sometimes get this popup from linux desktop client. Not sure what is triggering this.

![image](https://user-images.githubusercontent.com/379266/156195154-fc8a44f4-4d84-4260-8c64-416429777b47.png)

**Comment by BlackDex** [comment]
Seems to be client warning it self and probably not something on the server side. Please see if this also happens on vault.bitwarden.com.
